app/main.py

```diff
@@ -1,13 +1,52 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+    return {
+        "errors":
+            [
+                {
+                    "line": errors[i]["line_number"],
+                    "column": errors[i]["column_number"],
+                    "message": errors[i]["text"],
+                    "name": errors[i]["code"],
+                    "source": "flake8"
+
+                } for i in range(len(errors))
```

Comment:
Consider using a list comprehension directly with the `errors` list instead of using `range(len(errors))`. This can make the code more readable and Pythonic. For example: `[{'line': error['line_number'], 'column': error['column_number'], 'message': error['text'], 'name': error['code'], 'source': 'flake8'} for error in errors]`.